src/components/shared/AutoSave/index.tsx

```diff
@@ -11,16 +11,15 @@ const AutoSave = ({ values, onSave, debounceDelay }: AutoSaveProps) => {
   // We also don't want to save when the component is unmounted, but a
   // save had already been invoked and pending the debounce delay.
   const isMounted = useRef(false);
-  const debouncedSave = useCallback(
+  const debouncedSave = useMemo(() => {
```

Comment:
The former use of `useCallback` was causing an error like this one: https://github.com/facebook/react/issues/18833